clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -100,6 +100,8 @@ Improvements to clang-tidy
 - Improved :program:`run-clang-tidy.py` script. Added argument `-source-filter`
   to filter source files from the compilation database, via a RegEx. In a
   similar fashion to what `-header-filter` does for header files.
+- Improved :program:`check_clang_tidy.py` script. Added argument `-export`
```

Comment:
Please merge with previous entry.

Reply:
Sorry, my bad - I missed that tool names are different :-( Please drop later commit.